modules/apps/commerce/commerce-test/src/testIntegration/java/com/liferay/commerce/internal/search/test/CommerceOrderIndexerTest.java

```diff
@@ -0,0 +1,279 @@
+			_commerceCurrency.getCommerceCurrencyId());
+
+		commerceOrder2.setBillingAddressId(randomAddress2.getAddressId());
+		commerceOrder2.setShippingAddressId(randomAddress2.getAddressId());
+
+		commerceOrder2 = _commerceOrderLocalService.updateCommerceOrder(
+			commerceOrder2);
+
+		commerceOrder2 = _commerceOrderEngine.checkoutCommerceOrder(
+			commerceOrder2, _user.getUserId());
+
+		_commerceOrderLocalService.updateCommerceOrder(commerceOrder2);
+
+		Country country = randomAddress1.getCountry();
+
+		_assertSearch(country.getA2(), commerceOrder1);
+		_assertSearch(country.getName(), commerceOrder1);
+
+		_assertSearch(randomAddress1.getCity(), commerceOrder1);
+		_assertSearch(
+			randomAddress1.getExternalReferenceCode(), commerceOrder1);
+		_assertSearch(randomAddress1.getName(), commerceOrder1);
+		_assertSearch(randomAddress1.getStreet1(), commerceOrder1);
+		_assertSearch(randomAddress1.getStreet2(), commerceOrder1);
+		_assertSearch(randomAddress1.getStreet3(), commerceOrder1);
+		_assertSearch(randomAddress1.getZip(), commerceOrder1);
+
+		Region region = randomAddress1.getRegion();
+
+		_assertSearch(region.getName(), commerceOrder1);
+	}
+
+	private Address _addRandomAddress() throws Exception {
+		Country country = CommerceInventoryTestUtil.addCountry(_serviceContext);
+
+		Region region = CommerceInventoryTestUtil.addRegion(
+			country.getCountryId(), _serviceContext);
+
+		return _addressLocalService.addAddress(
+			RandomTestUtil.randomString(), _user.getUserId(),
+			AccountEntry.class.getName(), _accountEntry.getAccountEntryId(),
+			country.getCountryId(), 0, region.getRegionId(),
+			RandomTestUtil.randomString(), RandomTestUtil.randomString(), false,
+			RandomTestUtil.randomString(), true, RandomTestUtil.randomString(),
+			RandomTestUtil.randomString(), RandomTestUtil.randomString(), null,
+			RandomTestUtil.randomString(), RandomTestUtil.randomString(),
+			_serviceContext);
+	}
+
+	private void _assertSearch(
+			Hits hits, CommerceOrder... expectedCommerceOrders)
+		throws Exception {
+
+		List<CommerceOrder> actualCommerceOrders = _getCommerceOrders(hits);
+
+		long[] actualCommerceOrderIds = _getCommerceOrderIds(
+			actualCommerceOrders);
+
+		long[] expectedCommerceOrderIds = _getCommerceOrderIds(
+			Arrays.asList(expectedCommerceOrders));
+
+		Assert.assertArrayEquals(
+			expectedCommerceOrderIds, actualCommerceOrderIds);
+	}
+
+	private void _assertSearch(
+			String keywords, CommerceOrder... expectedCommerceOrders)
+		throws Exception {
+
+		SearchContext searchContext = _getSearchContext();
+
+		searchContext.setKeywords(keywords);
+
+		Hits hits = _indexer.search(searchContext);
+
+		_assertSearch(hits, expectedCommerceOrders);
+	}
+
+	private CommerceOrder _getCommerceOrder(Document document)
+		throws Exception {
+
+		long commerceOrderId = GetterUtil.getLong(
+			document.get(Field.ENTRY_CLASS_PK));
+
+		return _commerceOrderLocalService.getCommerceOrder(commerceOrderId);
+	}
```

Comment:
is it possible to inline this? I mean... it's used only once and are just two lines